clang/lib/CodeGen/CodeGenModule.h

```diff
@@ -1537,6 +1537,12 @@ class CodeGenModule : public CodeGenTypeCache {
   void printPostfixForExternalizedDecl(llvm::raw_ostream &OS,
                                        const Decl *D) const;
 
+  /// Under debug mode, print status of target teams loop transformation,
+  /// which should be either '#distribute' or '#parallel for'
+  void emitTargetTeamsLoopCodegenStatus(std::string StatusMsg,
+                                        const OMPExecutableDirective &D,
+                                        bool IsDevice);
+
```

Comment:
Don't like the idea of adding new member function specifically for debugging

Reply:
> Don't like the idea of adding new member function specifically for debugging

Agreed. Changed it to a static local function.